pom.xml

```diff
@@ -1149,6 +1149,12 @@
       </build>
     </profile>
 
+    <profile>
+      <id>hadoop-provided</id>
+      <properties>
+        <hadoop.scope>provided</hadoop.scope>
```

Comment:
Is this scope needed to be added here? https://github.com/logicalclocks/incubator-livy/blob/228cbe21fd18e8815a1882889146ab5040b2f43b/pom.xml#L336

Reply:
Yes it's needed. It overwrites the scope here https://github.com/logicalclocks/incubator-livy/blob/branch-0.6/server/pom.xml#L122 which is the jars we are interested.